### ISSUE TITLE
Inaccurate info in README

### ISSUE THREAD

**Issue by dev-dan** [ethereum/meteor-package-tools#3]
Krakken API no longer used but still referenced in README. 
Doc's detail usage of unit references that are not utilized in the code. Am not clear if these were deprecated or are net yet implemented.

**Comment by frozeman** [comment]
Yes we now use crypto compares API, can you change the text to them?

**Comment by dev-dan** [comment]
Sure! I will update the API Info now.

Want about the units in the README.md?

It states - `btc`,  `usd`,  `eur`, `cad`, `gbp`, `jpy` are possible for some methods, but as far as I can see only usd, eur, and btc are enabled through supportedCurrencies, and only usd/btc is supported through the current cryptocompare api call. 

Should the text of support for those units be removed until the code is extended to support the additional units or is there something I missed that facilitates these units?
